docs/advanced_topics/embeds.rst

```diff
@@ -191,6 +191,40 @@ For example, this is how you can instruct Youtube to return videos in HTTPS
     Wagtail will not try to run any other finder, even if the chosen one didn't
     return an embed.
 
+.. _Instagram:
+
+Facebook and Instagram
+----------------------
+
+As of October 2020, Facebook deprecated their public oEmbed APIs. If you would
+like to embed Facebook or Instagram posts in yoru site, you will need to
+use the new authenticated APIs. This requires you to set up a Facebook
+Developer Account and create a Facebook App that includes the oEmbed Product.
+Instructions for creating the neccessary app are in requirements sections of the
+`Facebook <https://developers.facebook.com/docs/plugins/oembed>`_
+and `Intagram <https://developers.facebook.com/docs/instagram/oembed>`_ documentation.
+
+Once you have your app access tokens (App ID and App Secret), add the Facebook and/or
+Instagram finders to your ``WAGTAILEMBEDS_FINDERS`` setting and configure them with
+the App ID and App Secret from your app:
+
+.. code-block:: python
+
+    WAGTAILEMBEDS_FINDERS = [
+        {
+            'class': 'wagtail.embeds.finders.facebook',
+            'app_id': 'YOUR FACEBOOK APP_ID HERE',
+            'app_secret': 'YOUR FACEBOOK APP_SECRET HERE',
```

Comment:
maybe also document the `omitscript` setting?

Reply:
Originally I had documented it but to be honest, I wasn't clear what it did so left it available but as an undocumented feature.

Reply:
Mh good point. It's not necessarily something that is easy to explain. Let me give this a try…

>>>
By default, Facebook includes some JavaScript that is necessary to fully render the embed.

In certain cases, this might not be something you want. For example, if you have multiple Facebook embeds, this would result in multiple script tags. So you can specify `omitscript` and Facebook won't include any script tags in the embed HTML. Note that you will then have to take care of loading this script yourself.
>>>

How's that?